src/main/java/net/earthcomputer/clientcommands/mixin/commands/generic/ChatScreenMixin.java

```diff
@@ -19,4 +22,9 @@ private String onHandleChatInput(String message) {
         }
         return command;
     }
+
+    @Inject(method = "init", at = @At("TAIL"))
+    private void onInit(CallbackInfo ci) {
+        ReplyCommand.onChatOpened();
```

Comment:
This should probably be in a separate mixin in the `mixin.commands.reply` package

Reply:
okie dokie artichokie